Client/src/Pages/Settings/index.jsx

```diff
@@ -1,9 +1,978 @@
+                      label="Update"
+                      onClick={handleUpdatePicture}
+                      sx={{
+                        height: "fit-content",
+                        color: theme.palette.primary.main,
+                        fontSize: "13px",
+                        "&:focus": {
+                          outline: "none",
+                        },
+                      }}
+                    />
+                  </Stack>
+                ) : (
+                  ""
+                )}
+              </div>
+            ))}
+          </form>
+          <Divider aria-hidden="true" sx={{ marginY: theme.spacing(6.25) }} />
+          <form className="delete-profile-form" noValidate spellCheck="false">
+            <div className="delete-profile-form__wrapper">
+              <Stack direction="column" gap="15px">
+                <Typography variant="h4" component="h1">
+                  Delete account
+                </Typography>
+                <Typography variant="h5" component="p">
+                  Note that deleting your account will remove all data from our
+                  system. This is permanent and non-recoverable.
+                </Typography>
+                <Box sx={{ mt: theme.spacing(1) }}>
+                  <Button
+                    level="error"
+                    label="Delete account"
+                    onClick={() => setIsOpen(true)}
+                    sx={{
+                      fontSize: "13px",
+                      "&:focus": {
+                        outline: "none",
+                      },
+                    }}
+                  />
+                </Box>
+              </Stack>
+            </div>
+          </form>
+          <Divider
+            aria-hidden="true"
+            width="0"
+            sx={{ marginY: theme.spacing(6.25) }}
+          />
+          {/* !!! - All save buttons are tied to the same state */}
+          {/* TODO - Implement Save Profile function */}
+          <Stack direction="row" justifyContent="flex-end">
+            <Box width="fit-content">
+              <ButtonSpinner
+                level="primary"
+                label="Save"
+                onClick={handleSaveProfile}
+                isLoading={isLoading}
+                loadingText="Saving..."
+                sx={{
+                  paddingX: "40px",
+                  height: "fit-content",
+                  fontSize: "13px",
+                  "&:focus": {
+                    outline: "none",
+                  },
+                }}
+              />
+            </Box>
+          </Stack>
+        </TabPanel>
+        <TabPanel
+          value="1"
+          sx={{ padding: "0", marginTop: theme.spacing(6.25), width: "100%" }}
+        >
+          <form className="edit-password-form" noValidate spellCheck="false">
+            <div className="edit-password-form__wrapper">
+              <AnnouncementsDualButtonWithIcon
```

Comment:
Required prop is missing here:

```
index.jsx:439 Warning: Failed prop type: The prop `closeToast` is marked as required in `AnnouncementsDualButtonWithIcon`, but its value is `undefined`.
```

Reply:
I believe that error is caused by not passing a toast close function. Will likely be fixed when Monzer finishes updating the toast component.

Reply:
Got it